modules/apps/commerce/commerce-frontend-js/package.json

```diff
@@ -23,6 +23,7 @@
 		"@clayui/tooltip": "3.4.5",
 		"@liferay/frontend-js-react-web": "*",
 		"classnames": "2.2.6",
+		"d3": "^5.9.2",
```

Comment:
Was going to ask why you chose this version seeing as [d3 is at 6.7.0](https://www.npmjs.com/package/d3), but... `yarn why d3` on `master` tells me (edited for brevity):

```
=> Found "d3@5.9.2"
info Has been hoisted to "d3"
info Reasons this module exists
   - Hoisted from "_project_#portal-workflow-metrics-web#@clayui#charts#d3"
   - Hoisted from "_project_#frontend-taglib-chart#clay-charts#clay-charts-shared#d3"
   - Hoisted from "_project_#portal-workflow-metrics-web#@clayui#charts#billboard.js#d3"
=> Found "commerce-organization-web#d3@5.0.0"
info This module exists because "_project_#commerce-organization-web" depends on it.
=> Found "clay-charts#d3@5.0.0"
info This module exists because "_project_#frontend-taglib-chart#clay-charts" depends on it.
```

Reply:
I started from this version only because when i started working on this i saw the only module in LR that was using d3 had that version. But i'll try to update to the last version crossing all my finger

Reply:
I don't think you actually have to update. Using an existing version (that is already in DXP) is fine as long as you aren't missing some new feature of bugfix.